examples/c++-api/Makefile

```diff
@@ -0,0 +1,7 @@
+all: count-stuff
+
+clean:
+	-rm count-stuff 
+
+count-stuff: count-stuff.cc
+	$(CXX) -I ../../lib count-stuff.cc -o count-stuff ../../lib/liboxli.a
```

Comment:
The `liboxli.a` file isn't created when I compile on OS X El Capitan. I see some `.so` files but these are for the wrong architecture.

I don't currently have access to my Ubuntu sandbox, so spinning up an Ubuntu VM for testing now.